general_considerations.md

```diff
@@ -241,6 +337,17 @@ can be delivered to an exchange, market maker, or OTC desk in exchange for
 immediate payment (over Lightning, for example) or as part of a derivatives
 contract.
 
+[kp]
+
+This is underestimating the complexity involved. Who proposes the exchange/swap,
+who accepts it. and finally how do they accept it. Are we going to need another
+consensus protocol or a co-ordinator here?
+
+In any case, if we go with the approach of supporting transactions, we need to
+spec them too.
+
+[/kp]
+
```

Comment:
who and how the swap is traded is outside the scope of Braidpool. We don't want to build a DEX here. All Braidpool needs to know is that there's a new address controlling the shares, and that address may have a script, but this is decided privately among the parties swapping shares for BTC.